src/a4/domain/IncomeTaxSpace.java

```diff
@@ -0,0 +1,21 @@
+package a4.domain;
+
+public class IncomeTaxSpace extends BoardSpace {
+	public IncomeTaxSpace() {
+	}
+
```

Comment:
Will we just set the value after creating the board space? When is this value set?